Dockerfile_firelens

```diff
@@ -1,5 +1,4 @@
-# We can't go past 1.20.X until this issue is solved: https://github.com/golang/go/issues/62130#issuecomment-1687335898
-FROM golang:1.20.7-bullseye AS builder
+FROM golang:1.23.6-bullseye AS builder
```

Comment:
I was afraid of doing this change, but it seems this does no longer happen. The CI pipeline built the Firelens image and ran it against a Mockserver and it sent logs correctly. 👍